src/messages/aggregate/publish.ts

```diff
@@ -42,41 +46,45 @@ type AggregatePublishConfiguration<T> = {
  *
  * @param configuration The configuration used to publish the aggregate message.
  */
-export async function Publish<T>(configuration: AggregatePublishConfiguration<T>): Promise<AggregateMessage<T>> {
+export async function Publish<T>({
+    account,
+    address,
+    key,
+    content,
+    channel,
+    storageEngine = ItemType.inline,
```

Comment:
Can the default not be configured in the AggregatePublishConfiguration type in TS ?